README.md

```diff
@@ -153,12 +157,24 @@ Pjax.prototype.getElements = function() {
   return document.getElementsByClassName(".js-Pjax")
 }
 
-new Pjax({})
+new Pjax()
 ```
 
-When instantiating a `Pjax` object, you need to pass all options as an object:
+#### `loadUrl(href, (options))`
```

Comment:
Should `(options)` be `[options]` to signify it's optional?

Reply:
That's what the extra brackets are for, but will change to the JSDoc syntax.